src/providers/push/pubnub/pubnub.provider.ts

```diff
@@ -20,6 +26,56 @@ export class PubNubProvider implements Provider<PubNubNotification> {
   }
 
   pubnubService: Pubnub;
+  getGeneralMessageObject(message: PubNubMessage) {
+    const commonDataNotification: MessageConfig = Object.assign(
+      {
+        title: message.subject,
+        description: message.body,
+      },
+      message.options,
+    );
+    const pnGcm: PnGcm = {
+      data: commonDataNotification,
```

Comment:
??, this does not solve the issue raised.

Reply:
now it is  resolved